src/containers/UserProfile/EditIntro.scss

```diff
@@ -49,3 +50,8 @@
     }
   }
 }
+.backdrop {
```

Comment:
@Humnaa is it required? Why are we adding a custom style for backdrop?

Reply:
same reason.. forgot to remove..